lib/ssusysinfo.c

```diff
@@ -52,6 +54,21 @@
  */
 #define EXPECTED_SSU_CONFIG_VERSION 14
 
+/** Primary path for OS release data */
+#define OS_RELEASE_PATH1        "/etc/sailfish-release"
```

Comment:
We should not use /etc/sailfish-release, but /etc/os-release only as it is symbolic link to the actual release file.

Reply:
For future proofing we should first parse  /usr/lib/os-release if /etc/os-release doesn't exist.
Read:
https://www.freedesktop.org/software/systemd/man/os-release.html

Reply:
Changed. Also I assumed that the future proofing applies for hw-release too.